novactf/protocols/protocol_tomoReconstruction.py

```diff
@@ -382,7 +402,7 @@ def _validate(self):
         validateMsgs = []
         ts = self.getInputTs()
 
-        if self.applyAlignment and not ts.getFirstItem().getFirstItem().hasTransform():
+        if not ts.getFirstItem().getFirstItem().hasTransform():
```

Comment:
if you are removing applyAlignment param, the protocol will fail for unaligned tilt-series.

Reply:
Shoudn't we trust the hasAlignment() method in the set of tilt series. And shouldn't this be enough to distinguish this case?

Reply:
sure, it make sense to use hasAlignment()